recipe/conda_build_config.yaml

```diff
@@ -678,8 +678,7 @@ root_base:
 ruby:
   - 2.5
   - 2.6
-r_base:
-  - 4.1
+r_base:   # [not win]
```

Comment:
When dropping 4.1, we need to skip windows entirely. Not sure how to do that.

Reply:
I guess we could stick with 4.1 for windows still. That'd bypass the issue.